src/test/shared/utilities/cliUtils.test.ts

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+import { makeTemporaryToolkitFolder } from '../../../shared/filesystemUtilities'
+import { Logger } from '../../../shared/logger'
+import { makeLogger } from '../../../shared/logger/activation'
+import { WinstonToolkitLogger } from '../../../shared/logger/winstonToolkitLogger'
+import { installCli } from '../../../shared/utilities/cliUtils'
+import { FakeWindow } from '../vscode/fakeWindow'
+import { TestSettingsConfiguration } from '../../utilities/testSettingsConfiguration'
+
+describe('cliUtils', async function () {
+    let tempFolder: string
+    let testLogger: Logger | undefined
+
+    const settingsConfig = new TestSettingsConfiguration()
```

Comment:
This also doesn't do anything? It's creating a new object and not passed anywhere.